tap-agent/src/agent/sender_allocation.rs

```diff
@@ -34,6 +43,55 @@ use crate::{
     tap::{context::checks::AllocationId, escrow_adapter::EscrowAdapter},
 };
 
+lazy_static! {
+    static ref UNAGGREGATED_FEE_PER_SENDER_X_ALLOCATION: GaugeVec = register_gauge_vec!(
+        format!("unagregated_fee_per_sender_x_allocation"),
+        "Unggregated Fees",
+        &["sender", "allocation"]
+    )
+    .unwrap();
+}
+
+lazy_static! {
+    static ref RAV_VALUE: GaugeVec =
+        register_gauge_vec!(format!("rav_value"), "RAV Updated value", &["allocation"]).unwrap();
+}
+
+lazy_static! {
+    static ref CLOSED_ALLOCATIONS: Counter = register_counter!(
+        format!("amount_of_closed_allocations"),
+        "allocations closed",
+    )
+    .unwrap();
+}
```

Comment:
I'd be more specific as to what this is counting:
```suggestion
lazy_static! {
    static ref CLOSED_SENDER_ALLOCATIONS: Counter = register_counter!(
        format!("closed_sender_allocation_total"),
        "Total count of sender-allocation managers closed.",
    )
    .unwrap();
}
```